circle.yml

```diff
@@ -159,6 +159,18 @@ jobs:
           command: |
             codecov --required --gcov-exec "$GCOV" --gcov-root ~/build
 
+  gcc-32bit:
+    executor: linux
+    environment:
+      CMAKE_OPTIONS: -DTOOLCHAIN=cxx17-32bit
+    steps:
+      - run:
+          name: "Install GCC 32-bit"
+          command: |
+            sudo apt -q update && sudo apt -qy install g++-multilib && sudo apt -qy install gcc-multilib
```

Comment:
I don't think you need to install `gcc-multilib` when `g++-multilib` is installed.

Reply:
done.